packages/manager/apps/vrack-services/src/public/translations/vrack-services/listing/Messages_fr_FR.json

```diff
@@ -17,5 +17,6 @@
   "BHS": "Canada, Beauharnois (BHS)",
   "vrackServicesCreateSucessMessage": "vRack Services creation success",
   "vrackCreateSucessMessage": "vRack creation success",
-  "vrackCreateErrorMessage": "Un problème est survenue lors de la commande de votre vRack. Veuillez réessayer ultérieurement."
+  "vrackCreateErrorMessage": "Un problème est survenue lors de la commande de votre vRack. Veuillez réessayer ultérieurement.",
```

Comment:
That's a bit too long
Why would they do this ?

Reply:
Because this is how it was in the previous angular version, which seems to be a convention, [Have a look here](https://github.com/ovh/manager/blob/master/packages/manager/modules/vrack/src/dashboard/translations/Messages_en_GB.json)